sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_operations/_patch.py

```diff
@@ -0,0 +1,327 @@
+T = TypeVar("T")
+ClsType = Optional[Callable[[PipelineResponse[HttpRequest, HttpResponse], T, Dict[str, Any]], Any]] # type: ignore[misc]
+
+
+class DocumentTranslationClientOperationsMixin(GeneratedDocumentTranslationClientOperationsMixin):
+
+    @distributed_trace
+    def begin_start_translation( # type: ignore[override]
+        self, body: Union[_models.StartTranslationDetails, JSON, IO[bytes]], **kwargs: Any
+    ) -> DocumentTranslationLROPoller[_models.TranslationStatus]:
+        _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+        _params = kwargs.pop("params", {}) or {}
+
+        content_type: Optional[str] = kwargs.pop("content_type", _headers.pop("Content-Type", None))
+        cls: ClsType[_models.TranslationStatus] = kwargs.pop("cls", None)
+        polling: Union[bool, PollingMethod] = kwargs.pop("polling", True)
+        lro_delay = kwargs.pop("polling_interval", self._config.polling_interval)
+        cont_token: Optional[str] = kwargs.pop("continuation_token", None)
+        if cont_token is None:
+            raw_result = self._start_translation_initial( # type: ignore[func-returns-value]
+                body=body, content_type=content_type, cls=lambda x, y, z: x, headers=_headers, params=_params, **kwargs
+            )
+        kwargs.pop("error_map", None)
+
+        def get_long_running_output(pipeline_response):
+            response_headers = {}
+            response = pipeline_response.http_response
+            response_headers["Operation-Location"] = self._deserialize(
+                "str", response.headers.get("Operation-Location")
+            )
+
+            deserialized = _deserialize(_models.TranslationStatus, response.json())
+            if cls:
+                return cls(pipeline_response, deserialized, response_headers)
+            return deserialized
+
+        path_format_arguments = {
+            "endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, "str", skip_quote=True),
+        }
+
+        if polling is True:
+            polling_method: PollingMethod = cast(
+                PollingMethod, LROBasePolling(lro_delay, path_format_arguments=path_format_arguments, **kwargs)
+            )
+        elif polling is False:
+            polling_method = cast(PollingMethod, NoPolling())
+        else:
+            polling_method = polling
+        if cont_token:
+            return DocumentTranslationLROPoller[_models.TranslationStatus].from_continuation_token(
+                polling_method=polling_method,
+                continuation_token=cont_token,
+                client=self._client,
+                deserialization_callback=get_long_running_output,
+            )
+        return DocumentTranslationLROPoller[_models.TranslationStatus](
+            self._client, raw_result, get_long_running_output, polling_method
+        )
+
+
+class SingleDocumentTranslationClientOperationsMixin(
```

Comment:
What did we need to patch in this client?

Reply:
I think we will patch this operation for the preview release and review it again during GA